src/result.ts

```diff
@@ -27,4 +27,10 @@ export enum ErrorCode {
     CREATEPROFFESIONAL_FACILITYIDS_REQUIRED = 'CREATEPROFFESIONAL_FACILITYIDS_REQUIRED',
     UPDATEPROFFESIONAL_FACILITYIDS_REQUIRED = 'UPDATEPROFFESIONAL_FACILITYIDS_REQUIRED',
     INTERNAL_SERVER_ERROR = 'INTERNAL_SERVER_ERROR',
+    INVALID_ORDERBY_OPTION = 'INVALID_ORDERBY_OPTION',
+    ORDERBY_OPTION_NOT_FOUND = 'ORDERBY_OPTION_NOT_FOUND',
+    ORDERBY_OPTION_MAX_LIMIT = 'ORDERBY_OPTION_MAX_LIMIT',
+    ORDERBY_FIELD_SELECTED_TWICE = 'ORDERBY_FIELD_SELECTED_TWICE',
+    INVALID_LANGUAGE_INPUT = 'INVALID_LANGUAGE_INPUT',
+    SUSPICIOUS_PATTERN_DETECTED = 'SUSPICIOUS_PATTERN_DETECTED'
```

Comment:
Security lesson! 
Don't let them know that you've detected suspicious code. Then they can test around it and try to find code that IS NOT detected. 
Just return back an invalid input and let's log this suspicious attempt